.github/workflows/test.yaml

```diff
@@ -1094,6 +1094,7 @@ jobs:
     concurrency: dataproxy-${{ matrix.platform }}-${{ matrix.clientEngine }}
     env:
       FLAVOR: ${{ matrix.flavor }}
+      PRISMA_GENERATE_FLAG: ${{ matrix.flavor == 'DP1' && '--data-proxy' || '--accelerate' }}
```

Comment:
That would not really be necessary, as we know they work exactly the same. 

But: Nice.